eclair-core/src/main/scala/fr/acinq/eclair/json/JsonSerializers.scala

```diff
@@ -510,7 +510,6 @@ object ShortIdsSerializer extends ConvertClassSerializer[ShortIds](s => ShortIds
 // @formatter:off
 private case class FundingTxStatusJson(status: String, tx: Option[Transaction])
 object FundingTxStatusSerializer extends ConvertClassSerializer[LocalFundingStatus]({
-  case LocalFundingStatus.UnknownFundingTx => FundingTxStatusJson("unknown", None)
   case s: LocalFundingStatus.UnconfirmedFundingTx => FundingTxStatusJson("unconfirmed", s.signedTx_opt)
   case s: LocalFundingStatus.ConfirmedFundingTx => FundingTxStatusJson("confirmed", s.signedTx_opt)
```

Comment:
nit: I think we should only put the `txid` in the json serialization, otherwise it's potentially costly to serialize big transactions repeatedly (since we'll serialize this at every `storing()`).

Reply:
Done in [3a7b561](https://github.com/ACINQ/eclair/pull/2598/commits/3a7b5615f8fa2f16b87d23bdb983ed57a40e5102).